rotkehlchen/api/rest.py

```diff
@@ -6376,3 +6381,53 @@ def get_unmatched_asset_movements(self) -> Response:
         return api_response(_wrap_in_ok_result(
             result=list({event.group_identifier for event in asset_movements}),
         ))
+
+    def get_matches_for_asset_movement(
+            self,
+            asset_movement_group_identifier: str,
+            time_range: int,
+    ) -> Response:
+        events_db = DBHistoryEvents(database=self.rotkehlchen.data.db)
+        asset_movement = fee_event = None
+        with self.rotkehlchen.data.db.conn.read_ctx() as cursor:
+            for event in events_db.get_history_events_internal(
+                cursor=cursor,
+                filter_query=HistoryEventFilterQuery.make(group_identifiers=[asset_movement_group_identifier]),
+            ):
+                if isinstance(event, AssetMovement):
+                    if event.event_subtype == HistoryEventSubType.FEE:
+                        fee_event = event
+                    else:
+                        asset_movement = event
+
+        if asset_movement is None:
+            return api_response(wrap_in_fail_result(
+                message=f'No asset movement event found in the DB for group identifier {asset_movement_group_identifier}',  # noqa: E501
+            ), HTTPStatus.BAD_REQUEST)
+
+        close_match_identifiers = [x.identifier for x in find_asset_movement_matches(
+            events_db=events_db,
+            asset_movement=asset_movement,
+            is_deposit=asset_movement.event_type == HistoryEventType.DEPOSIT,
+            fee_event=fee_event,
+            match_window=time_range,
+        )]
+
+        asset_movement_timestamp = ts_ms_to_sec(asset_movement.timestamp)
+        with self.rotkehlchen.data.db.conn.read_ctx() as cursor:
+            other_events = events_db.get_history_events_internal(
+                cursor=cursor,
+                filter_query=HistoryEventFilterQuery.make(
+                    from_ts=Timestamp(asset_movement_timestamp - time_range),
+                    to_ts=Timestamp(asset_movement_timestamp + time_range),
+                    ignored_ids=[
+                        str(asset_movement.identifier),
+                        *[str(x) for x in close_match_identifiers],
+                    ],
```

Comment:
isn't this more redeable and better since you build only once?
```suggestion
                    ignored_ids=[
                     str(x) for x in close_match_identifiers,
                    ] + [str(asset_movement.identifier)],
```